src/components/forms/generic/RadioButtonFormComponentBroad.tsx

```diff
@@ -0,0 +1,156 @@
+import type { ChangeEvent } from 'react';
+import type { Control, FieldError, FieldValues, Path } from 'react-hook-form';
+import { useController } from 'react-hook-form';
+import type { FormControlLabelProps } from '@mui/material';
+import {
+	FormControl,
+	FormControlLabel,
+	FormHelperText,
+	FormLabel,
+	Radio,
+	RadioGroup,
+	useTheme
+} from '@mui/material';
+import { useFormError } from 'react-hook-form-mui';
+
+export type RadioButtonGroupProps<T extends FieldValues> = {
+	options: { label: string; id: string | number }[] | any[];
+	helperText?: string;
+	name: Path<T>;
+	required?: boolean;
+	parseError?: (error: FieldError) => string;
+	label?: string;
+	labelKey?: string;
+	valueKey?: string;
+	type?: 'number' | 'string';
+	emptyOptionLabel?: string;
+	onChange?: (value: any) => void;
+	returnObject?: boolean;
+	row?: boolean;
+	control?: Control<T>;
+	labelProps?: Omit<FormControlLabelProps, 'label' | 'control' | 'value'>;
+	disabled?: boolean;
+};
+
+export const RadioButtonFormComponentBroad = <
```

Comment:
Celý tenhle kód je udělaný podle [zdrojáku té knihovny](https://github.com/dohomi/react-hook-form-mui/blob/master/packages/rhf-mui/src/RadioButtonGroup.tsx), bohužel ta knihovní komponenta neumožňuje dostatečnou customizaci (nejde např. cokoliv ohledně labelu a ohraničení - stylovat jde jen buttony a jejich labely).
Máme dvě možnosti, co mě napadají:

- použít variantu RadioButtonFormComponent s omezenými možnostmi customizace, tzn. label a ohraničení dodat jinak (label si asi představit dokážu, ohraničení moc netuším) nebo se na ně vykašlat
- použít tuhle variantu (třída se sufixem `Broad`) a dodělat její customizaci (přesně podle figmy to ale pořád úplně neumím s tím labelem v ohraničení :D)